src/components/NotificationPanel/styles.css

```diff
@@ -0,0 +1,30 @@
+.NotificationPanel {
```

Comment:
when you scroll, the overlay covers up the notification panel
![image](https://user-images.githubusercontent.com/12878275/58135049-c1beb300-7bdd-11e9-99fa-77cf1d9d1c7f.png)